### DISCUSSION TITLE
Please explain how to find a monitor

### DISCUSSION THREAD

**Discussion by licopper** [linuxhw/EDID#11] [Show and tell]
Hi!

> The structure of the repository is the following:
{TYPE}/{VENDOR}/{MODEL}/{ID}
( e.g. Digital/LG Display/LGD0217/925C880E8A08 )
ID of a monitor is **MD5 of EDID**.

Please explain step by step how I can calculate the MD5 of a monitor's EDID.

**Comment by linuxhw** [comment]
Hi,

Why you need this? MD5 of EDID is needed only to find EDID file in the directory. You take it from the table in the README.

**Reply by linuxhw** [reply]
If you are still interested, it's just `md5sum` of the hex representation of the EDID file.

**Comment by licopper** [comment]
Hi,
Thanks for the response.
I was trying to find an EDID that conforms to my monitor's model in your DB.
Also I've uploaded my HW report to LHW Project but I don't see my EDID in
your database. Just trying to understand how it works.
If it's just MD5 of hex representation of the EDID file then I don't
understand how you get just 5 chars string as result. Please elaborate a
bit more.

**Reply by linuxhw** [reply]
Full-length md5 doesn't fit the page width on GitHub, so it's truncated to 5 chars prefix.

What is your monitor model name?

**Reply by linuxhw** [reply]
Also please post your probe ID.

**Comment by licopper** [comment]
My monitor is AOC Q27P1(B).
How do I get my probe ID?
thanks for replying.

On March 12, 2021 7:19:43 AM UTC, Linux Hardware Project ***@***.***> wrote:
>Also please post your probe ID.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/linuxhw/EDID/discussions/11#discussioncomment-473041

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Reply by linuxhw** [reply]
Usually hw-probe returns it after creating the probe:

    sudo -E hw-probe -all -upload

**Reply by linuxhw** [reply]
```
git clone https://github.com/linuxhw/EDID.git
cd EDID
grep -nR Q27P1
Digital/AOC/AOC2701/4CE5319347D6:69:    Display Product Name: 'Q27P1B'
Digital/AOC/AOC2701/D2DBF0481E0A:71:    Display Product Name: 'Q27P1B'
Digital/AOC/AOC2701/39CF2A1B3115:69:    Display Product Name: 'Q27P1B'
Digital/AOC/AOC2701/3852B513000A:71:    Display Product Name: 'Q27P1B'
Digital/AOC/AOC2701/787C40CCB549:71:    Display Product Name: 'Q27P1B'
Digital/AOC/AOC2701/3ED84CE4A2A0:71:    Display Product Name: 'Q27P1B'
Digital/AOC/AOC2701/5CE441B42759:69:    Display Product Name: 'Q27P1B'
```

**Comment by licopper** [comment]
Got it :)

I just found my probe ID here:
~# cat HW_PROBE/LOG
PROBE
=====
Date: Fri Feb 26 10:22:49 2021 (1614327769)
Probe URL: https://linux-hardware.org/?probe=7ca92abd15

Thank you very much! You helped to deal with my monitor/GPU setup issues.

**Comment by jackson3355** [comment]
hey just wanted to know if anyone can send me the edid of  my monitor: | Acer             | ACR075A | VG240Y V         | 1920x1080 | 24.0 | 2020 | [6B3EB7E11F15](<Digital/Acer/ACR075A/6B3EB7E11F15>) |
